src/scss/trumps/_utilities.scss

```diff
@@ -112,6 +120,10 @@
     padding-left: spacing(x4);
 }
 
+.u-noUnderline {
+    text-decoration: none;
```

Comment:
It's probably worth this helper having `:hover` and `:focus` state which applies the underline

Reply:
Done.